R/contr_one_hot.R

```diff
@@ -24,9 +25,14 @@ contr_one_hot <- function(n, contrasts = TRUE, sparse = FALSE) {
   }
 
   if (is.character(n)) {
+    check_character(n, empty = FALSE)
+    if (length(n) < 1) {
+      cli::cli_abort("A character vector for {.arg n} cannot be empty.")
```

Comment:
```suggestion
      cli::cli_abort("{.arg n} cannot be empty.")
```

via https://style.tidyverse.org/errors.html#problem-statement